### ISSUE TITLE
Usability and accessibility problems with remove block modal dialog

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#648]
The mechanism and dialog for removing a block has several usability and accessibility problems:

![Screenshot from 2023-01-05 19-31-02](https://user-images.githubusercontent.com/1008395/210857087-b599dde4-3e84-495b-b741-99781e437a1e.png)

- When opening the dialog, the dialog should be focused. Not something in the background.
- Tabbing through elements, I should not be able to leave the dialog unless I close it. As you can see in the screenshot, I'm able to tab somewhere in the background. This is confusing as a regular user and a big problem when using a screen reader.
- While I can tab to the “Remove block” button, I cannot tab to the “X” button to close the dialog
- “Are you sure?” implies a choice and an answer of “yes” or ”no” (or something with the same meaning) but you only get one button. I suggest adding a “Cancel” button
- “Are you sure?” does not explain what this dialog is about. Even though it is the header of this dialog. I suggest using a title which explains what will happen when you confirm this, like “Delete this block?”

For a list of best practices, take a look at [ARIA: dialog role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role)
